app/views/layouts/_navigation.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <a class="navbar-brand hide-desktop" href="/">
       <%= image_tag "mobile-brand.png", width: 30, height: 30, class: "d-inline-block align-text-top" %>
-      ScenicOrNot
+      <span>Scenic</span><span>Or</span><span>Not</span>
```

Comment:
Is this change related to word wrap?

Reply:
It's so we can style the each word in different colours.